jsk_mbzirc_tasks/launch/jsk_mbzirc_task_2.launch

```diff
@@ -14,8 +15,8 @@
   <arg name="cheat" default="false" />
 
   <!-- Spawn simulated ugv -->
-  <arg unless="$(arg cheat)" name="robot_initial_pose" value="-x 65  -y -25 -z 0.3" />
```

Comment:
The z value is a little bit high compare to the ground, and the  then I change it to 0.1m